.github/workflows/python-spark2.4-tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Python Tests for Spark 2.4
+
+on:
+  push:
+    branches:
+      - '4.0'
+      - 'master'
+  pull_request:
+    branches:
+      - '*'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
```

Comment:
it would be great add tests over different neo4j version, like here:
https://github.com/neo4j-contrib/neo4j-spark-connector/blob/422bf30486fb0c40f638631c12b89cdc17eadb57/.github/workflows/ci.yml#L16